task/grype/0.1/README.md

```diff
@@ -0,0 +1,36 @@
+# grype
+
+This task allows the use of the grype for Tekton Pipelines.See <https://github.com/anchore/grype>
```

Comment:
```suggestion
This task allows the use of the grype for Tekton Pipelines. See <https://github.com/anchore/grype>
```

Reply:
Could just drop this link since there's a "For more details, please see" below

Reply:
thanks